data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+				accept
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive', her novel about humans stealing jump drives from the Quarg. She hastily opens the hatch as soon as it unlocks.`
+			choice
+				`	"Wait for me!"`
+			`	Anna hears your scream and stops. "We're here, Anna. Now, we need to find their offices.", you say.`
+			`	"That won't be necessary. In the message they sent me, they included precise instructions on how to reach their offices."`
+			`	You begin following Anna to the publisher's offices. You walk through endless rows of shops and streets, take a train to a large white building, and eventually find what looks like the Kimek equivalent of an elevator.`
+			`	You and Anna enter the elevator, as Anna presses a button that closes the door and takes you up. When the door opens, you see a Kimek in front of you, a necklace around it with ornate jewelry and a translation device.`
+			`	"Greetings, humans. Follow me, you must, so publish your book, we can.", says the Kimek through its translation box.`
+			`	At the direction of the Kimek, you and Anna begin walking until directed to a room with glass walls. Anna puts down the copy of her book on the table, and the three of you take a seat.`
+			choice
+				`	"Hello. My name is <first> <last>, and this is my friend Anna Fangvin."`
+			`	"Greetings. At Kimek Reader Publishing, I work. A human, I have never encountered. But the book you sent, translate it we were able to."`
+			`	"Read it, we still haven't, and run it past the Heliarch censors, we must."`
+			`	Two Heliarch agents enter the room. "Supervise the interview, we are obligated to."`
+			choice
+				`	"How long will I have to wait?"`
+					goto "how long"
+				`	"Should I stay here to accompany Anna?`
+					goto "should I accompany anna"
+			label "how long"
+			`	"Wait a few hours, you must. But quick, this will be.", says the Kimek, interviewing you.`
+			choice
+				`	"Should I stay here to accompany Anna?"`
+			label "should I accompany anna"
+			`	"To accompany her in this room, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
+			choice
+				`	(Wait in the lounge.)`
+					goto "lounge"
+				`	(Accompany Anna during the interview.)`
+			`	You choose to remain in the room. Once the Kimek asks the first question, Anna is all too happy to begin discussing the contents of her novel in great detail.`
+			choice
+				`	(Leave the room to avoid spoilers.)`
+					goto "lounge"
+				`	(Stay in the room.)`
+			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, the content of the book. She is still talking in fluent English, but sometimes is forced to repeat statements due to the translator having issues with deciphering her enthusiastic tone.`
```

Comment:
The Coalition translators' issue lies with the sentence structure, not being incapable of translating or keeping up. I'd simply remove the mention of her needing to repeat herself due to translators' not keeping up, maybe replace it with mention that the Heliarchs occasionally keep pressing for further explanations on certain parts of the book, before letting her continue what she was saying.

Reply:
```suggestion
			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Haliarchs' numerous questions for clarification and elaboration make the process a long one.`
```